code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -301,7 +301,8 @@ Pipelines + Other Objects -> Pipe network
 #define VENT_SOUND_DELAY 30
 
 /obj/machinery/atmospherics/relaymove(mob/living/user, direction)
-	if(!(direction & initialize_directions)) //cant go this way.
+	direction &= initialize_directions
+	if(!direction || !(direction in GLOB.cardinals)) //cant go this way.

```

Comment:
the check for `!direction` is redundant

Reply:
if it's 0 as a result that'll short circuit and it's harmless otherwise